src/forms/reduxForm/__snapshots__/reduxTextInput.test.jsx.snap

```diff
@@ -0,0 +1,472 @@
+                      "unregister": [Function],
+                      "unregisterField": [Function],
+                      "untouch": [Function],
+                      "updateSyncErrors": [Function],
+                      "updateSyncWarnings": [Function],
+                      "valid": true,
+                      "validExceptSubmit": true,
+                      "values": Object {},
+                      "warning": undefined,
+                    }
+                  }
+                  _value="Wonder Woman and Robin"
+                  asyncValidating={false}
+                  component={[Function]}
+                  dirty={false}
+                  dispatch={[Function]}
+                  error="Did you mean Batman and Robin?"
+                  label="Super Hero"
+                  maxLength={20}
+                  name="superhero"
+                  normalize={[Function]}
+                  pristine={true}
+                  required={true}
+                >
+                  <ReduxTextInput
+                    error="Did you mean Batman and Robin?"
+                    input={
+                      Object {
+                        "name": "superhero",
+                        "onBlur": [Function],
+                        "onChange": [Function],
+                        "onDragStart": [Function],
+                        "onDrop": [Function],
+                        "onFocus": [Function],
+                        "value": "",
+                      }
+                    }
+                    label="Super Hero"
+                    maxLength={20}
+                    meta={
+                      Object {
+                        "active": false,
+                        "asyncValidating": false,
+                        "autofilled": false,
+                        "dirty": false,
+                        "dispatch": [Function],
+                        "error": undefined,
+                        "form": "testForm",
+                        "initial": undefined,
+                        "invalid": false,
+                        "pristine": true,
+                        "submitFailed": false,
+                        "submitting": false,
+                        "touched": false,
+                        "valid": true,
+                        "visited": false,
+                        "warning": undefined,
+                      }
+                    }
+                    required={true}
+                  >
+                    <TextInput
+                      error="Did you mean Batman and Robin?"
+                      label="Super Hero"
+                      maxLength={20}
+                      name="superhero"
+                      onBlur={[Function]}
+                      onChange={[Function]}
+                      onDragStart={[Function]}
+                      onDrop={[Function]}
+                      onFocus={[Function]}
+                      required={true}
+                      value=""
```

Comment:
There's a lot going on in this snapshot, so it's hard to tell where the issue is coming from - I'd recommend testing the `<TextInput />` in isolation and see if it's still not getting the `value` passed through.